cms/core/migrations/0006_update_user_groups.py

```diff
@@ -0,0 +1,192 @@
+from django.conf import settings
+from django.db import migrations
+from wagtail.models import PAGE_PERMISSION_CODENAMES, GroupCollectionPermission, GroupPagePermission
+
+WAGTAIL_PERMISSION_TYPES = ["add", "change", "delete"]
+
+WAGTAIL_PAGE_PERMISSION_TYPES = [codename.replace("_page", "") for codename in PAGE_PERMISSION_CODENAMES]
```

Comment:
let's go for the layman's approach for this 

`WAGTAIL_PAGE_PERMISSION_TYPES = ["add", "change", "bulk_delete", "lock", "publish", "unlock"]`